ios/FluentUI/List/FluentList.swift

```diff
@@ -44,12 +44,14 @@ public struct FluentList<ListContent: View>: View {
             case .insetGrouped:
                 list
                     .listStyle(.insetGrouped)
+                    .scrollContentBackground(.hidden)
```

Comment:
how come inset doesn't need?

Reply:
Inset doesn't seem to have a custom background color at the moment, so there's no need to remove the background.

@alexanderboswell, @nodes11, can either of you speak to why `.inset` exists but has no customization?

Reply:
Gonna merge for now but follow up offline :)

Reply:
I'm not exactly sure. The styles we had an internal use case for were added and customizations were going to be added as needed.